wd_helper.au3

```diff
@@ -961,7 +980,20 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 		Else
 			$sReadyState = _WD_ExecuteScript($sSession, 'return document.readyState', '', Default, $_WD_JSON_Value)
 			$iErr = @error
-			If $iErr Or $sReadyState = 'complete' Then
+			If $iErr Then
+				$iErr = $_WD_ERROR_Exception
+				$sReadyState = ''
+			Else
+				For $i = 0 To $_WD_READYSTATE__COUNTER - 1
+					If $sReadyState = $aWD_READYSTATE[$i][0] Then ; https://www.w3schools.com/jsref/prop_doc_readystate.asp
+						$iExt = $i
```

Comment:
This link doesn't belong here IMO.

Reply:
Fixed: [w3schools link](https://github.com/Danp2/au3WebDriver/pull/384/commits/9b5b2b5bb53355fb15695741275297bf6d2b311f)